fixtures/tracks/animal/img/icon.svg

```diff
@@ -0,0 +1,303 @@
+<!DOCTYPE html>
```

Comment:
Download this file: https://upload.wikimedia.org/wikipedia/commons/4/4b/Pictograms-nps-winter-ice_fishing-2.svg